dashboard/test/ui/features/step_definitions/blockly.rb

```diff
@@ -227,6 +227,94 @@
   @browser.execute_script("__TestInterface.loadBlocks('" + xml + "');")
 end
 
+And /^I've initialized the workspace with the artist_autorun blocks$/ do
```

Comment:
I know there are already a few specific solutions in this file, but I wonder if it would make sense to make a new `step_definitions` file that just has the steps for specific blockly puzzle setups. We can also add a comment to the top of that file explaining how you get the solution blocks for a new test.